src/types/product.ts

```diff
@@ -0,0 +1,21 @@
+export interface ProductType {
```

Comment:
We can create a BaseProductType containing the duplicated values with CartProductType.

Reply:
I was actually thinking of a way to get rid of the duplicate values and this solution fits well.